### PR TITLE
Improvement skeleton.

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: kit/layout/src/Skeleton/Skeleton.tsx [hunk 1]
```diff
@@ -213,6 +229,25 @@ export function UISkeleton({
...
+         */
+        ...Platform.select({
+            web: {
+                top: -1,
+                bottom: -1,
+                left: -1,
+                right: -1,
+            },
+            android: {
+                top: -0.5,
```
Comment (savelichalex) [inline kit/layout/src/Skeleton/Skeleton.tsx]: Actually seems to be fragile in devices with pixel density = 1

File: casts/rows/src/UILink/UILink.tsx [hunk 2]
```diff
@@ -59,6 +59,7 @@ const styles = StyleSheet.create({
     },
     textContent: {
         flex: 1,
+        borderRadius: 0,
```
Comment (savelichalex) [inline casts/rows/src/UILink/UILink.tsx]: What this for? AFAIR it's 0 by default. What is it doing?
Need to at least leave a comment, to not delete it along the way, as it seems unnecessary without the context.

Comment (SAnatoliiS) [reply casts/rows/src/UILink/UILink.tsx]: Removed, no longer required

File: kit/layout/src/Skeleton/Skeleton.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply kit/layout/src/Skeleton/Skeleton.tsx]: I agree, this solution has many drawbacks. Deleted. Used `opacity: 0` for children

File: kit/layout/src/Skeleton/Skeleton.tsx [hunk 3]
```diff
@@ -193,18 +199,21 @@ export function UISkeleton({
...
-            style={[
-                styles.container,
-                { borderRadius: isVisible ? UILayoutConstant.alertBorderRadius : 0 },
-                styleProp,
-            ]}
-            onLayout={onLayout}
-        >
-            {children}
+        <View style={[styles.container, styleProp]} onLayout={onLayout}>
+            <View style={{ opacity: show ? 0 : 1 }}>{children}</View>
```
Comment (savelichalex) [inline kit/layout/src/Skeleton/Skeleton.tsx]: Does it mean that `Skeleton` isn't behave like a regular `View` anymore? For example setting `flexDirection: row` for it won't have any effect on content.

Comment (SAnatoliiS) [reply kit/layout/src/Skeleton/Skeleton.tsx]: You're right, I'm going back to the previous solution with a little revision

**Review by savelichalex — APPROVED** [review]

File: kit/layout/src/Skeleton/Skeleton.tsx [hunk 4]
```diff
@@ -213,7 +222,13 @@ export function UISkeleton({
 const styles = StyleSheet.create({
     container: {
         position: 'relative',
+    },
+    skeletonContainer: {
         overflow: 'hidden',
+        top: -StyleSheet.hairlineWidth,
+        left: -StyleSheet.hairlineWidth,
+        bottom: -StyleSheet.hairlineWidth,
+        right: -StyleSheet.hairlineWidth,
```
Comment (sertony) [inline kit/layout/src/Skeleton/Skeleton.tsx:237]: That looks weird to me... Can you add a comment why do we need it?
Also, it looks like it's better to add `position: 'absolute'` here in order to ensure that the applied style contains it (without relying on `StyleSheet.absoluteFill`)

**Review by sertony — APPROVED** [review]

**Pull request merged**